src/styles/main.scss

```diff
@@ -1 +1,72 @@
-// put code here
+* {
+  box-sizing: border-box;
+}
+
+.body {
```

Comment:
Use 'body' instead of '.body'. Dot notation is used for classes, not for elements.